packages/@react-spectrum/s2/src/TableView.tsx

```diff
@@ -784,20 +790,29 @@ function ResizableColumnContents(props: ResizableColumnContentProps) {
           </div>
           <Chevron size="M" className={chevronIcon} />
         </Button>
-        <Menu onAction={onMenuSelect} items={items} styles={style({minWidth: 128})}>
-          {(item) => <MenuItem>{item?.label}</MenuItem>}
+        <Menu onAction={onMenuSelect} styles={style({minWidth: 128})}>
+          {items.length > 0 && (
+            <MenuSection aria-label="Sort or resize?">
```

Comment:
not translated. also probably shouldn't be a question

Reply:
haha, it's a question because https://github.com/adobe/react-spectrum/pull/7617#discussion_r1921677519